Jenkinsfile_CNP

```diff
@@ -198,15 +182,6 @@ withPipeline(type, product, component) {
     steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'test-results/smokeTest/**/*'
   }
 
-  before('functionalTest:aat') {
-    env.NODE_TLS_REJECT_UNAUTHORIZED = "0";
-    env.IDAM_API_URL = "https://idam-api.aat.platform.hmcts.net"
-    env.SERVICE_AUTH_PROVIDER_API_BASE_URL = "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
-    env.CCD_DATA_STORE_URL = "https://civil-ccd-data-store-staging-aat.service.core-compute-aat.internal"
-    env.DM_STORE_URL = "http://dm-store-aat.service.core-compute-aat.internal"
-    env.CIVIL_SERVICE_URL="https://civil-ccd-staging.service.core-compute-aat.internal"
```

Comment:
@adusumillipraveen  where are we setting this ? the master pipeline requires this for the tests to run against the staging pods . . the before functionaltest:preview is fine since you have moved this to onPR tag @mounikahmcts thoughts ?

Reply:
They are all set in before(smoketest:aat) Its just removing duplication.